docs/_pages/documentation.md

```diff
@@ -980,6 +980,18 @@ With trace:
   }
   ```
 
+  By default, trace is displayed only when assertion fails as part of the message. If the trace is needed for successful assertion you can catch it like this:
+
+  ```csharp
+object object1 = [...];
+object object2 = [...];
+var traceWriter = new StringBuilderTraceWriter();            
+object1.Should().BeEquivalentTo(object2, opts => opts.WithTracing(traceWriter));             
+string trace = traceWriter.ToString();
+  ```
+
+Alternatively, you could write your own implementation of ITraceWriter for special purposes e.g. writing to a file.
```

Comment:
🙃 I would use back-tickets for the reference to `ITraceWriter` so it looks like code.

Reply:
Good catch, thanks